mkdocs.yml

```diff
@@ -67,112 +73,142 @@ extra_javascript:
-    - Msgport Interfaces: 'msgport/interfaces.md'
-    - Msgport Workflow: 'msgport/workflow.md'
-    - Msgport API: 'msgport/api.md'
-    - Msgport Scan: 'msgport/scan.md'
-    - Messaging Protocols:
-        - Overview: 'msgport/messaging-protocols/overview.md'
-        - ORMP: 'msgport/messaging-protocols/ormp.md'
-        - XCMP: 'msgport/messaging-protocols/xcmp.md'
-        - LCMP: 'msgport/messaging-protocols/lcmp.md'
-    - Supported Network: 'msgport/networks.md'
-    - User Cases:
-        - Overview: 'msgport/user-cases/overview.md'
-        - Order XClearing: 'msgport/user-cases/order-xclearing.md'
-        - XToken: 'msgport/user-cases/xtoken.md'
-        - XAccount: 'msgport/user-cases/xaccount.md'
-        - XDAO: 'msgport/user-cases/xdao.md'
-    - Tutorial:
-        - Pangolin > Sepolia Remix Demo: 'msgport/tutorial/remix-demo.md'
-        - Pangolin > Sepolia Script Demo: 'msgport/tutorial/script-demo.md'
-    - FAQ: 'msgport/faq.md'
-    - TroubleSome: 'msgport/troublesome.md'
-  - EVM+: 
-    - Overview: 'evm/overview.md'
-    - Chains And Explorers:
-          - Overview: 'evm/chains/overview.md'
-          - Darwinia Chain: 'evm/chains/darwinia.md'
-          - Crab Chain: 'evm/chains/crab.md'
-          - Pangolin Chain: 'evm/chains/pangolin.md'
-          - Rollup Testnet: 'evm/chains/rollup.md'
-    - Available Wallets: 'evm/wallets.md'
-    - Ethereum Compatibility:
-          - Account System: 'evm/ethereum-compatibility/account-system.md'
-          - JSON-RPC API: 'evm/ethereum-compatibility/jsonrpc.md'
-          - Gas Compatibility: 'evm/ethereum-compatibility/gas.md'
-          - EVM Tracing: 'evm/ethereum-compatibility/evm-tracing.md'
+      - Overview: "msgport/overview.md"
+      - Glossary: "msgport/glossary.md"
+      - Interfaces: "msgport/interfaces.md"
+      - Workflow: "msgport/workflow.md"
+      - API Documentation: "msgport/api.md"
+      - Message Scan: "msgport/scan.md"
+      - Messaging Protocols:
+          - Overview: "msgport/messaging-protocols/overview.md"
+          - ORMP Protocol: "msgport/messaging-protocols/ormp.md"
+          - XCMP Protocol: "msgport/messaging-protocols/xcmp.md"
+          - LCMP Protocol: "msgport/messaging-protocols/lcmp.md"
+      - Supported Networks: "msgport/networks.md"
+      - Use Cases:
+          - Overview: "msgport/use-cases/overview.md"
+          - Order Clearing: "msgport/use-cases/order-xclearing.md"
+          - XToken Management: "msgport/use-cases/xtoken.md"
+          - XAccount Operations: "msgport/use-cases/xaccount.md"
+          - XDAO Functions: "msgport/use-cases/xdao.md"
+      - Tutorials:
+          - Using Remix on Pangolin > Sepolia: "msgport/tutorial/remix-demo.md"
+          - Script-Based Setup on Pangolin > Sepolia: "msgport/tutorial/script-demo.md"
+      - FAQs: "msgport/faq.md"
+      - Troubleshooting: "msgport/troublesome.md"
+  - EVM+:
+      - Overview: "evm/overview.md"
+      - Chains and Explorers:
```

Comment:
Why not And?

Reply:
Refer https://docs.soliditylang.org/en/v0.8.25/

ChatGPT:

> In English titles, conjunctions and prepositions are usually lowercase, except when they are the first word of the title or the last word. Words like "and", "or", "but", "on", "in", "at", and "by" are generally not capitalized in titles unless they appear at the beginning or the end. This is a common rule in English writing for capitalizing titles. If you need the rules for titles in a specific language, please let me know, and I can provide more specific assistance.